pyproject.toml

```diff
@@ -41,17 +41,22 @@ dependencies = [
 zip-safe = false
 
 [tool.pytest.ini_options]
-testpaths = ["test"]
+testpaths = ["src/*/test*"]
 addopts = "--verbose"
+pythonpath = ["src"]
 
 [tool.black]
 line-length = 88
 target-version = ["py39"]
 skip-string-normalization = true
 
 [tool.pylint]
-disable = ["W0511", "C0301"]
+disable = ["W0511", "C0301", "E1101", "E0237"]
```

Comment:
Just wondering, why are we disabling these rules?

Reply:
The way ROS requires us to setup the code base results in a few linting errors within Python. It's rather annoying and couldn't find a reasonable solution so this is like a short-term solution to get linting to pass.